### ISSUE TITLE
Adds extra points for already completed waypoint

### ISSUE THREAD

**Issue by tesla809** [FreeCodeCamp/FreeCodeCamp#3009]
Challenge http://freecodecamp.com/challenges/waypoint-import-a-google-font has an issue. Please describe how to reproduce it, and include links to screenshots if possible.

I noticed new waypoints for HTML5/CSS sections. I completed some of the new ones, and just to refreshen up on some older waypoints I finished a while ago, I decided to redo some. 

I noticed that I was receiving points for complete the waypoints that I have already done. These were the ones that were done before the new update in the curriculum.

**Comment by bugron** [comment]
@tesla809 big thanks for pointing this out!
I'm able to confirm this. I've just submitted [waypoint: Import a Google Font](http://freecodecamp.com/challenges/waypoint-import-a-google-font) and after that my Brownie points were incremented. Tried in FF 38.
cc @BerkeleyTrue, @benmcmahon100

**Comment by BerkeleyTrue** [comment]
I can't reproduce this. @bugron Are you still able to replicate? I've tried multiple times (I always test new things using the same three challenges and have not noticed this issue).

**Comment by awesomeaniruddh** [comment]
Can't reproduce this either: using the latest version of Chrome on a Windows 10 machine.

**Comment by BerkeleyTrue** [comment]
@awesomeaniruddh Thanks. I have a feeling hitting the back button and being served the browser cached version of the site is confusing campers.

**Comment by BerkeleyTrue** [comment]
@tesla809 Are you still experiencing this and can you reproduce it reliably?

**Comment by bugron** [comment]
@BerkeleyTrue I've just submitted [Waypoint: Import a Google Font](http://freecodecamp.com/challenges/waypoint-import-a-google-font) one more time but I haven't got a Brownie point. Then I've submitted [Waypoint: Specify How Fonts Should Degrade](http://www.freecodecamp.com/challenges/waypoint-specify-how-fonts-should-degrade) and I got a Brownie point and the same for next waypoint!

**Comment by allanarmstrong** [comment]
I've also experienced this issue. I've been going back and re-doing some old bonfires with more efficient code and I've noticed that I've been getting the extra points for ones I've already done. Using Chrome latest version. (chrome 45). 

![screen shot 2015-10-17 at 4 30 44 am](https://cloud.githubusercontent.com/assets/506160/10548672/1dac0f66-7488-11e5-9a29-a34fe296df39.png)
![screen shot 2015-10-17 at 4 30 56 am](https://cloud.githubusercontent.com/assets/506160/10548673/1dac4fda-7488-11e5-87a8-444a1cb2df1c.png)
![screen shot 2015-10-17 at 4 31 13 am](https://cloud.githubusercontent.com/assets/506160/10548674/1dad99f8-7488-11e5-9d8e-bf1ea163e8c9.png)
